ios/sandwich/QonversionSandwich.swift

```diff
@@ -13,53 +13,53 @@ public class QonversionSandwich : NSObject {
   
   // MARK: - Private variables
   
-  private var qonversionEventListener: QonversionEventListener
+  private var qonversionEventListener: QonversionEventListener? = nil
```

Comment:
Should it really be set (and thus become nullable) even if it is initialized via constructor?

Reply:
It's an artifact of `NSObject` inheritance that is required for `Obj-c` compatibility. Another way is to use force unwrap, but I think optional is better even though we know that the listener will be initialized.